hrs-portlets-webapp/src/main/webapp/WEB-INF/jsp/timeAbsence.jsp

```diff
@@ -88,25 +88,25 @@
             <table class="dl-table table">
               <thead>
                 <tr rsf:id="header:">
-                  <th class="flc-pager-sort-header" rsf:id="name"><a href="javascript:;">Name</a></th>
-                  <th class="flc-pager-sort-header" rsf:id="status"><a href="javascript:;">Status</a></th>
-                  <th class="flc-pager-sort-header" rsf:id="start"><a href="javascript:;">Start</a></th>
-                  <th class="flc-pager-sort-header" rsf:id="end"><a href="javascript:;">End</a></th>
-                  <th class="flc-pager-sort-header" rsf:id="total"><a href="javascript:;">Total</a></th>
+                  <th scope="col" class="flc-pager-sort-header" rsf:id="name"><a href="#">Name</a></th>
```

Comment:
Does having the `<a>` in there break the sorting mechanism that is in fluid sorting?